plugin/documents.py

```diff
@@ -490,8 +491,9 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
                 title = '{} code actions'.format(action_count)
             else:
                 title = next(itertools.chain.from_iterable(responses.values()))['title']
+                title = os.linesep.join([title[:30], title[30:]])
```

Comment:
Why are you using `os.linesep`? In ST environment everything is pretty much normalized to `\n` and it should work with just that.

Reply:
Thank you for reminding me. I was not clear about it before, and I will revise it immediately

Reply:
Resolved in 0eed685